src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/decompositions.py

```diff
@@ -107,6 +107,7 @@ def get_aot_decomposition_list():
         torch.ops.aten._scaled_dot_product_flash_attention.default,
         torch.ops.aten._softmax.default,
         torch.ops.aten._softmax_backward_data.default,
+        torch.ops.aten._unsafe_index,
```

Comment:
Does this operator appear only when aot_autograd option is set to True? Can this be added to get_inf_decomposition_list instead?